### PR TITLE
Customize TrueLayer payments element with green

### PR DESCRIPTION
**Pull request by peschina**
https://github.com/ogcio/life-events/assets/47299026/68ff3d2e-d3be-444b-840b-1c839a7020fc

### CODE REVIEW

File: apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/OpenBankingHost.tsx [hunk 1]
```diff
@@ -21,12 +21,12 @@ export default function Page(props: Props) {
       resource_token: props.resourceToken,
       return_uri: props.returnUri,
       appearance: {
-        default_color: ds.colours.ogcio.darkTurquoise,
+        default_color: "#004d44",
```
Comment (peschina) [inline apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/OpenBankingHost.tsx:24]: This is the colour of our buttons. Should we add it to the design system colours?

Comment (alfonsograziano) [reply apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/OpenBankingHost.tsx:24]: I think so. The "green" on the DS is not the green of the button. We can have something like "ds.colours.primaryButton" I guess

Comment (peschina) [reply apps/payments/app/[locale]/(hosted)/paymentRequest/bankTransfer/OpenBankingHost.tsx]: [4c1e0fb](https://github.com/ogcio/life-events/pull/36/commits/4c1e0fbececbe1fa9cb4636fba32af3ae248dab3)

**Review by alfonsograziano — APPROVED** [review]
LGTM

**Pull request merged**